cdap-docs/admin-manual/source/operations/index.rst

```diff
@@ -81,31 +81,58 @@ Command Line Interface. See :ref:`reference:cli` in the
 
 .. highlight:: console
 
-Administrators can check the health of various services in the system.
-(In these examples, substitute for ``<host>`` the host name or IP address of the CDAP server.)
+Administrators can check the health of various services in the system. (In these examples,
+substitute for ``<host>`` the host name or IP address of the CDAP server. Substitute the
+correct ports if they have been changed from the default values shown.) On success, these
+calls return a valid HTTP response with a 200 code.
 
-- To retrieve the **health check of the CDAP UI**, make a GET request to the URI::
+- To retrieve the **health check of the CDAP UI**, make a GET request to the URI (at the
+  ``dashboard.bind.port``)::
 
     http://<host>:9999/status
+  
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:9999/status"
 
-- To retrieve the **health check of the CDAP Router**, make a GET request to the URI::
+
+- To retrieve the **health check of the CDAP Router**, make a GET request to the URI (at
+  the ``router.bind.port``)::
 
     http://<host>:10000/status
 
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:10000/status"
+
 - To retrieve the **health check of the CDAP Authentication Server**, make a GET request to
-  the URI::
+  the URI (at the ``security.auth.server.bind.port``)::
   
     http://<host>:10009/status
 
-On success, the calls return a valid HTTP response with a 200 code.
+  Such as (for the Standalone CDAP):
+   
+  .. tabbed-parsed-literal::
+
+    $ curl -w"\n" -X GET "http://localhost:10009/status"
 
-- To retrieve the **health check of all the services running in YARN**, make a GET request
-  to the URI::
+- To retrieve a **list of health checks of all the CDAP system services running in YARN**, make a GET request
```

Comment:
I could be wrong, but I thought this was to distinguish services such as CDAP Master, which are run outside YARN, right? But perhaps we need further clarification?